### DISCUSSION TITLE
Welcome to phantom-camera Discussions!

### DISCUSSION THREAD

**Discussion by ramokz** [ramokz/phantom-camera#56] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Janakanty** [comment]
Hey. Im from Polish Godot comunity and I'm creating a top-down horror game. When I saw this plugin, I knew right away that I wanted to use it. However, now I have a problem: I have a lot of random dynamic scenes. These scenes have a phantom camera with a path option. How can I dynamically add the player to the phantom camera?

Something like this is giving me an error:
area_pcam.follow_path = get_node("LevelController/Path2D")

Somebody could help me?

**Reply by ramokz** [reply]
Hello!

Good question, at the minute the ability to dynamically assign properties at runtime can be a bit inconsistent with the code like above.
One of the additions for v0.4 is better support for exactly cases like this however.

For the time being, you can add this to the bottom of the `phantom_camera_2D.gd` file:
```
func set_follow_path(value: Path2D) -> void:
	Properties.follow_path_node = value
```

Then you can dynamically apply a path with the method `area_pcam.set_follow_path(get_node("LevelController/Path2D"))`

If you then update to v0.4, or later, when it comes out then that should keep working as it'll be the exact same function code.

**Comment by emberlightstudios** [comment]
How can we preview the view from a virtual camera?

**Reply by ramokz** [reply]
Do you mean how you can look through the lens of the active PhantomCamera?

If so, there's a tab at the bottom of the Editor called `Phantom Camera`, which opens the viewfinder that shows the currently active `PhantomCamera` node.

**Reply by emberlightstudios** [reply]
Yep. Thanks

**Comment by ramokz** [comment]
Locking this post as it's meant as an introduction to the discussion section of the repo and to avoid questions being hidden away underneath that banner.

For new questions or other topics, please start a [new discussion](https://github.com/ramokz/phantom-camera/discussions/new/choose).
